bin/node/runtime/src/lib.rs

```diff
@@ -348,6 +351,7 @@ impl pallet_babe::Config for Runtime {
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
 	type EpochChangeTrigger = pallet_babe::ExternalTrigger;
+	type DisabledValidators = Session;
```

Comment:
A [fix](https://github.com/paritytech/substrate/pull/9414) which doesn't allow disabled validators to build blocks.